featuretools/tests/entityset_tests/test_entity.py

```diff
@@ -1,236 +0,0 @@
-from datetime import datetime
-
-# import numpy as np
-import pandas as pd
-import pytest
-import woodwork as ww
-
-import featuretools as ft
-# from featuretools.entityset import Entity, EntitySet
-from featuretools.tests.testing_utils import (
-    make_ecommerce_entityset,
-    to_pandas
-)
-from featuretools.utils.gen_utils import Library
-
-
-def test_is_index_column(es):
-    assert es['cohorts'].index == 'cohort'
-
-
-def test_reorders_index():
```

Comment:
I made a note of this in the comment of the main integration PR so we can hopefully document later.

Reply:
I do believe it's listed in the release notes' breaking changes `Featuretools will no longer re-order columns in DataFrames so that the index column is the first column of the DataFrame` but definitely good to include elsewhere as well

Reply:
Cool, sounds like this is resolved.